packages/xcoobee-cookie-kit-react/src/CookieKitContainer.js

```diff
@@ -174,8 +156,7 @@ export default class CookieKitContainer extends React.PureComponent {
     if (!isLoginStatusChecked) {
       this.setState({ isLoginStatusChecked: true });
       this.getCountryCode()
-        .then(() => this.getCookieConsents())
-        .catch(handleErrors);
+        .then(() => this.getCookieConsents());
```

Comment:
I am concerned that we are not handling possible errors here anymore. I believe `callCallbacks` is a possible source of errors. I think `this.getCountryCode` (unless used on a browser without `localStorage`) is safe and `this.getCookieConsents` is mostly safe. The unsafe part is the call to `this.fallBackToHostDefaults`. The call to `this.fallBackToHostDefaults` may call `this.setCookieConsents` which calls `this.callCallbacks` which may error.